R/cfbd_games.R

```diff
@@ -1363,7 +1363,8 @@ cfbd_game_team_stats <- function(year,
       }
       df <- df %>%
         tidyr::unnest(.data$teams) %>%
-        tidyr::unnest(.data$stats)
+        tidyr::unnest(.data$stats) %>%
+        dplyr::distinct()
```

Comment:
can you explain how this fixes the long-data issue for a R-layman

Reply:
Added a comment but the cause was an error in the pivoting to wide in the next line caused by rare duplicate values from CFBD. Distinct removes the duplicates allowing the rest of the code to run normally. BIll also fixed the duplicates from week 1 in CFBD already so it's not currently an issue, but this should present it popping up in the future